src/Sentry.Azure.Functions.Worker/Sentry.Azure.Functions.Worker.csproj

```diff
@@ -6,6 +6,11 @@
       <Description>Official Azure Functions Worker SDK integration for Sentry - Open-source error tracking that helps developers monitor and fix crashes in real time.</Description>
     </PropertyGroup>
 
+    <PropertyGroup Condition="'$(FrameworkSupportsAot)' == 'true'">
+      <IsAotCompatible>true</IsAotCompatible>
+      <EnableConfigurationBindingGenerator>true</EnableConfigurationBindingGenerator>
```

Comment:
Do we always set this when IsAotCompatible is true? If so, should we set this in the Directory.Build.Props where we define IsAotCompatible instead?

Reply:
We only set it for NuGet packages that we intend to support AOT. At the moment that's:

- [ ] Sentry.csproj
- [ ] Sentry.AspNetCore.csproj
- [ ] Sentry.Azure.Functions.Worker.csproj
- [ ] Sentry.Extensions.Logging.csproj
- [ ] Sentry.Maui.csproj
- [ ] Sentry.NLog.csproj
- [ ] Sentry.OpenTelemetry.csproj
- [ ] Sentry.Profiling.csproj
- [ ] Sentry.Serilog.csproj

Still missing are:

- [ ] Sentry.Android.AssemblyReader.csproj
- [ ] Sentry.AspNet.csproj (N/A Legacy)
- [ ] Sentry.AspNetCore.Grpc.csproj (TODO?)
- [ ] Sentry.Bindings.Android.csproj (???)
- [ ] Sentry.Bindings.Cocoa.csproj (???)
- [ ] Sentry.DiagnosticSource.csproj (N/A Legacy?)
- [ ] Sentry.EntityFramework.csproj (N/A Legacy)
- [ ] Sentry.Google.Cloud.Functions.csproj (TODO?)
- [ ] Sentry.Log4Net.csproj (N/A Legacy)

I think this is related to https://github.com/getsentry/sentry-dotnet/issues/2890. Once everything that targets net8.0 is AotCompatible, we could put this in the `Directory.Build.props` but I don't think we can do that yet (and not in this PR).

Reply:
Also we can add the `false` explicitly where needed